Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -56,27 +54,29 @@ - (instancetype)initWithFrame:(CGRect)frame {
 - (void)layoutSubviews {
     [super layoutSubviews];
 
-    CGFloat padding = (CGRectGetWidth(self.bounds) - CGRectGetWidth(self.shapeLayer.bounds)) / 2.0 - 0.5;
-    if (padding > CGRectGetWidth(self.shapeLayer.bounds) / 8.0) {
-        padding = CGRectGetWidth(self.shapeLayer.bounds) / 8.0;
-    }
+    CGFloat titleHeight = 31;
```

Comment:
Can we move these magic numbers to file level constants instead of variables in the method?